app/modules/entities/components/cleanup/lib/add_pertinance_score.ts

```diff
@@ -0,0 +1,45 @@
+import leven from 'leven'
+import { min, flatten, intersection, values } from 'underscore'
+import type { SerializedEntity } from '#app/modules/entities/lib/entities'
+import { getSerieOrWorkExtendedAuthorsUris } from '#app/modules/entities/lib/types/serie_alt'
+
+interface WorkSuggestionExtras {
+  pertinanceScore: number
+  authorMatch: boolean
+  labelMatch: boolean
+}
+
+export type WorkSuggestion = SerializedEntity & WorkSuggestionExtras
+
+export function addPertinanceScore (serie: SerializedEntity) {
```

Comment:
Pertinence (with a "e") is hardly used on a day 2 day basis. Relevant is much more [frequent](https://www.linguee.fr/francais-anglais/search?source=auto&query=pertinence)
```suggestion
export function addRelevantScore (serie: SerializedEntity) {
```

Reply:
done in 8ab032f18